creator-node/src/services/sync/initBootstrapAndRefreshPeers.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
copied from `processSync.js` with very slight modifications:
- rename `redisKey` to `logPrefix`
- wrpa in try-catch